.kokoro/client/main.go

```diff
@@ -0,0 +1,39 @@
+package main
```

Comment:
I think we don't need a client. The benchmarking code will have a client. (so, I think we can remove this directory)